beacon-chain/state/state-native/getters_validator.go

```diff
@@ -273,3 +265,15 @@ func (b *BeaconState) InactivityScores() ([]uint64, error) {
 
 	return b.inactivityScoresVal(), nil
 }
+
+// inactivityScoresVal of validators participating in consensus on the beacon chain.
+// This assumes that a lock is already held on BeaconState.
+func (b *BeaconState) inactivityScoresVal() []uint64 {
```

Comment:
I guess the Val pattern doesn't need to check versions because it will be nil if version is incorrect?

Reply:
I was going back and forth with this one... I think we do want to check versions and return `nil` in the case of mismatch.